qml/harbour-berlin-vegan.qml

```diff
@@ -56,29 +52,17 @@ ApplicationWindow
         model: gJsonVenueModel
         currentPosition: globalPositionSource.position.coordinate
     }
-    
-    function openDataBase() {
-        db = LocalStorage.openDatabaseSync("BerlinVeganDB", "0.1", "Berlin Vegan SQL!", 1000000);
-        db.transaction(function(tx) {
-            tx.executeSql("CREATE TABLE IF NOT EXISTS BerlinVegan(favorite_id TEXT)");
-            favorite_ids = tx.executeSql("SELECT favorite_id FROM BerlinVegan");
-        });
-    }
-
-    function applyFavoritesFromDataBase()
```

Comment:
Just wondering: Is there a specific reason why inlined this back into the ```favoritesHook()```

Reply:
You mean the `favourite_ids` part? I could not access `gJsonVenueModel` directly and it just seemed wrong the pass it into a `BVApp.Database.dbXYZ()` function.

Reply:
Yes, agreed.
But we could leave it here? But that's a minor…

Reply:
Then please merge as is.